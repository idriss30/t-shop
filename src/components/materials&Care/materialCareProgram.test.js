import React from "react";
import MaterialCare from "./materialCareProgram";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

test("matching materialComponent snapshot", () => {
  expect(
    render(
      <MemoryRouter>
        <MaterialCare />
      </MemoryRouter>
    )
  ).toMatchSnapshot();
});
