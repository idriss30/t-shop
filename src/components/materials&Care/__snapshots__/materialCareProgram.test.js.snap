// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matching materialComponent snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="css-1yqcee9-MaterialCare"
      >
        <div
          class="css-849ju6-MaterialCare"
        >
          <div
            class="css-1sr3x1s-MaterialCare"
          >
            <h1
              class="css-1cqwli6-MaterialCare"
            >
              Content & Care
            </h1>
            <h2
              class="css-q9heeh-MaterialCare"
            >
              Join Life
            </h2>
            <p>
              Care for water: produced using less water.
            </p>
            <p>
              We use the Join Life label for items that have been produced using technologies and raw materials that help us reduce the environmental impact of our products.
            </p>
            <p>
              These garments were produced using technologies that reduce water consumption in their production processes.
            </p>
          </div>
          <div
            class="css-1qx9l4v-MaterialCare"
          >
            <a
              href="/shop/zipBlack"
            >
              <img
                alt="black tee-shirt"
                class="css-u5131a-MaterialCare"
                src="zipBlack-display.jpg"
              />
            </a>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="css-1yqcee9-MaterialCare"
    >
      <div
        class="css-849ju6-MaterialCare"
      >
        <div
          class="css-1sr3x1s-MaterialCare"
        >
          <h1
            class="css-1cqwli6-MaterialCare"
          >
            Content & Care
          </h1>
          <h2
            class="css-q9heeh-MaterialCare"
          >
            Join Life
          </h2>
          <p>
            Care for water: produced using less water.
          </p>
          <p>
            We use the Join Life label for items that have been produced using technologies and raw materials that help us reduce the environmental impact of our products.
          </p>
          <p>
            These garments were produced using technologies that reduce water consumption in their production processes.
          </p>
        </div>
        <div
          class="css-1qx9l4v-MaterialCare"
        >
          <a
            href="/shop/zipBlack"
          >
            <img
              alt="black tee-shirt"
              class="css-u5131a-MaterialCare"
              src="zipBlack-display.jpg"
            />
          </a>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
